package.json

```diff
@@ -184,6 +186,7 @@
         "react-native-svg-transformer": "^1.0.0",
         "react-refresh": "^0.10.0",
         "react-test-renderer": "18.2.0",
+        "rxjs": "^7.8.1",
```

Comment:
can remove this

Reply:
I forgot this was resolving the `Cannot find module 'rxjs/operators'` error in the RequestInterceptor test